src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -120,13 +120,15 @@ function TransactionRequestContent({ request, onConfirm, onReject }: Props) {
         {t('walletConnect.requests.transactionRequest')}
       </Text>
 
-      <LargeFiatTokenValueView
-        value={value}
-        assetAddress={assetAddress}
-        chain={chain}
-        symbol={symbol}
-        style={styles.tokenValue}
-      />
+      {!value?.isZero() && (
```

Comment:
In current code null value causes component to render (but then component renders itself to null), so maybe we could use a more straightforward condition.
```suggestion
      {value?.gt(0) && (
```

Reply:
Done